<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Simulador de matricula 3000</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblMostrador1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblMostrador2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblMostrador3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblEstudiante1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblEstudiante2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblEstudiante3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblEstudiante4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblEstudiante5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblEstudiante6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblEstudiante7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblEstudiante8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblEstudiante9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>410</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblEstudiante10">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>450</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReporteCarreras">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Carreras</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblReportes">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Reportes:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCursosDeUnaCarrera">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Cursos de una carrera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProfesores">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Profesores</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProfesoresDeUnCurso">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Profesores de un curso</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEstudianteDeUnGrupo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Estudiantes de un grupo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEstudiantesDeUnCarrera">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Estudiantes de una carrera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProfesoresDeServicio">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Profesores de servicio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProfesoresDeCarrera">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>450</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Profesores de carrera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCantidadDePersonas">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>500</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Personas atendidas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Carrera:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txfCarreraParaCursos">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Carrera:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txfCarreraParaProfesores">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>200</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txfCursosParaProfesores">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>200</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>200</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Curso:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txfCarreraParaEstudiantes">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>250</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>250</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Curso:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Carrera:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txfCursosParaEstudiantes">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>250</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>250</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Grupo:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txfGrupoParaEstudiantes">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>250</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txfCarreraParaEstudiantes2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Carrera:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProfesoresDeUNACarrera">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Profesores de una carrera</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txfCarreraParaProfesores2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>350</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Carrera:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
